lib/packwerk/commands/help_command.rb

```diff
@@ -0,0 +1,27 @@
+# typed: strict
+# frozen_string_literal: true
+
+module Packwerk
+  module Commands
+    class HelpCommand < BaseCommand
+      extend T::Sig
+
+      sig { override.returns(T::Boolean) }
+      def run
+        @err_out.puts(<<~USAGE)
```

Comment:
Here too, I'm preserving existing behaviour: this help message was output to `err_out` before, still is.